app/gui/view/documentation/src/html.rs

```diff
@@ -0,0 +1,342 @@
+//! HTML generator for documentation.
+
+use enso_prelude::*;
+use horrorshow::prelude::*;
+
+use enso_suggestion_database::documentation_ir::Constructors;
+use enso_suggestion_database::documentation_ir::Documentation;
+use enso_suggestion_database::documentation_ir::EntryDocumentation;
+use enso_suggestion_database::documentation_ir::Function;
+use enso_suggestion_database::documentation_ir::ModuleDocumentation;
+use enso_suggestion_database::documentation_ir::Synopsis;
+use enso_suggestion_database::documentation_ir::TypeDocumentation;
+use enso_suggestion_database::engine_protocol::language_server::DocSection;
+use enso_suggestion_database::engine_protocol::language_server::Mark;
+use enso_suggestion_database::entry::Argument;
+use horrorshow::box_html;
+use horrorshow::labels;
+use horrorshow::owned_html;
+
+
+
+// =============
+// === Icons ===
+// =============
+
+/// We use SVG icons imported as text.
+type Icon = &'static str;
+const ICON_TYPE: &str = include_str!("../assets/icon-type.svg");
+const ICON_METHODS: &str = include_str!("../assets/icon-methods.svg");
+const ICON_EXAMPLES: &str = include_str!("../assets/icon-examples.svg");
+/// A value for `viewBox` attribute of the SVG icon. Depends on the size of the icon exported from
+/// Figma.
+const ICON_VIEWBOX: &str = "0 0 32 32";
+const ICON_SVG_XMLNS: &str = "http://www.w3.org/2000/svg";
+
+/// A single icon used in headers. `content` is an SVG code of the icon's content _without_ the
+/// surrounding `<svg>` tags.
+fn svg_icon(content: &'static str) -> impl Render {
+    let class = "w-5 h-5 fill-none flex-shrink-0 mt-0.5";
+    owned_html! {
+        svg(class=class, viewBox=ICON_VIEWBOX, xmlns=ICON_SVG_XMLNS) {
+            :Raw(content)
+        }
+    }
+}
+
+
+
+// ==============
+// === Render ===
+// ==============
+
+/// Render entry documentation to HTML code with Tailwind CSS styles.
+pub fn render(docs: EntryDocumentation) -> String {
+    match docs {
+        EntryDocumentation::Placeholder(_) => String::from("Temporary placeholder"),
```

Comment:
is it possible that someone ever sees this text?

Reply:
Shouldn't happen at the moment, but I'll improve it to avoid possible confusion.